### PR TITLE
fix: remove assertion for simulation completion from experiment entity api's calls

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**
Jira - https://modelonproducts.atlassian.net/browse/FLOW-544

### CODE REVIEW

**Review by efredriksson-modelon — APPROVED** [review]
Thanks for adding this, looks good to me :+1:

File: modelon/impact/client/entities/experiment.py [hunk 1]
```diff
@@ -413,19 +413,23 @@ def _validate_and_fetch_trajectories(
...
-        _assert_experiment_is_complete(self.run_info.status, "Simulation")
         assert_variable_in_result(variables, self.get_variables())
 
         return self._sal.experiment.trajectories_get(
             self._workspace_id, self._exp_id, variables, only_last_point, format.value
         )
 
-    def get_trajectories(self, variables: List[str]) -> Dict[str, Any]:
+    def get_trajectories(
+        self, variables: List[str], verify_simulation_is_complete: bool = True
```
Comment (iakovn) [inline modelon/impact/client/entities/experiment.py:423]: `verify_simulation_is_complete` is kind of long name. How about something shorter:
- verify_complete
- assert_complete
- check_simulation_done
- ensure_simulation_complete
- check_finished

Comment (abhilash-kumar-nair) [reply modelon/impact/client/entities/experiment.py:423]: I will go for ensure_simulation_complete :)

Comment (abhilash-kumar-nair) [reply modelon/impact/client/entities/experiment.py:423]: Or wait ensure sounds like we wait until simulation is complete.. "check_simulation_done" probably is the more appropriate

**Comment by abhilash-kumar-nair** [comment]
As discussed on teams, I have modified the code to remove checks for simulation completion from experiment entity api's.

**Pull request merged**